build/images/Dockerfile.controller.ubuntu

```diff
@@ -0,0 +1,23 @@
+# Copyright 2024 Antrea Authors
```

Comment:
Just curious to know why is this file added, when we already have dockerfile.build.controller.ubuntu.

Reply:
See comment above

Reply:
got it.